src/config/media/pipewire.md

```diff
@@ -0,0 +1,52 @@
+# PipeWire
+
+To use PipeWire, install the `pipewire` package.
+
+PipeWire should be started as a user. Use the autostarting mechanism of your
+desktop environment. If using [startx](../graphical-session/xorg.md#startx),
+add the following line in your `.xinitrc` file:
+
+```
+pipewire &
+```
+
+## PulseAudio replacement
+
+PipeWire can start a PulseAudio server. To enable it, uncomment the following
```

Comment:
I would now suggest to launch `pipewire-pulse` in the same place where you launch `pipewire`, instead of from the config. Keeping the config unmodified allows xbps to automatically replace it with the updated versions.

Reply:
That's a great suggestion.
I've tried this on Sway, by executing both `pipewire` and `pipewire-pulse` and it works fine.